components/server/src/authorization/spicedb-authorizer.ts

```diff
@@ -82,17 +78,14 @@ export class SpiceDBAuthorizer {
     }
 
     async deleteRelationships(req: v1.DeleteRelationshipsRequest): Promise<v1.ReadRelationshipsResponse[]> {
-        if (!this.client) {
-            return [];
-        }
-
         const timer = spicedbClientLatency.startTimer();
         let error: Error | undefined;
         try {
-            const existing = await this.client.readRelationships(v1.ReadRelationshipsRequest.create(req));
+            const client = this.clientProvider.getClient();
```

Comment:
we could extract this into 

`get client() {
   return this.clientProvider.getClient();
}